docs/data/charts/lines/lines.md

```diff
@@ -61,6 +61,21 @@ This property expects one of the following string values, corresponding to the i
 
 {{"demo": "InterpolationDemo.js", "hideToolbar": true, "bg": "inline"}}
 
+### Optimization
+
+To show mark elements, use `showMark` series property.
+It accepts a boolean or a callback.
+The next example shows how to use it to display only one mark every two data points.
+
+When the value is highlighted, an additional mark is rendered over the other ones.
```

Comment:
Isn't the current sentence a bit misleading?
Based on how it works, it seems that a mark is rendered only for a given value, which is being highlighted.
```suggestion
When a value is highlighted, a mark is rendered for that given value.
```

Reply:
Yes, but if you set `showMark=true` for N data points, you get N marks + the highlighted one

Reply:
Hmmm... Gotcha. 🤔 
Then maybe the explanation could be even more clear. 🤷

Reply:
What about 

```suggestion
When a value is highlighted, a mark is rendered for that given value.
If the charts already have some marks (due to `showMark=true`) the highlight one will be on top of others.
```

Reply:
IMHO, the suggestion brings more clarity. 👍 💯